tests/run-test-suite/src/libyaml-emitter.c

```diff
@@ -0,0 +1,253 @@
+#include <yaml.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <assert.h>
+
+bool get_line(FILE *input, char *line);
+char *get_anchor(char sigil, char *line, char *anchor);
+char *get_tag(char *line, char *tag);
+void get_value(char *line, char *value, int *style);
+
+int main(int argc, char *argv[]) {
```

Comment:
Removed Docker files and references to docker.

Reformatted code with GNU indent, using the style specified in the new `.indent.pro` file. Feel free to muck around with the settings. There seems to be hundreds of them.

You'll need to use the GNU version. This means that on Mac: `brew install gnu-indent` and then use `gindent` instead of `indent`.